core/modules/simpletest/tests/image.test

```diff
@@ -404,18 +404,6 @@ class ImageToolkitGdTestCase extends BackdropWebTestCase {
         $correct_dimensions_object = TRUE;
         $correct_colors = TRUE;
 
-        // PHP 5.5 GD bug: https://bugs.php.net/bug.php?id=65148. PHP 5.5 GD
-        // rotates differently then it did in PHP 5.4 resulting in different
-        // dimensions. For the test images, the dimensions will be 1 pixel
-        // smaller in both dimensions (though other tests have shown a
-        // difference of 0 to 3 pixels in both dimensions.
-        // @todo: The PHP bug was fixed in PHP 7.0.26 and 7.1.12. Change the code
-        //   below to reflect that in https://www.drupal.org/node/2918570.
-        if (version_compare(PHP_VERSION, '5.5', '>=') && $values['function'] === 'rotate' && $values['arguments'][0] % 90 != 0) {
```

Comment:
If I'm reading the comment correctly, then this bug exists from php5.5 all through 7.0.26 and 7.1.12. If that's the case, then we should leave this in place.

Reply:
I don't think the code comment is accurate. Our tests for PHP 7+ have always passed with this in place. It's possible that early versions of PHP 7.0.x and 7.1.x had the "old" calculation, but the latest releases of those versions pass with the current code. I think change in calculation happened mid-cycle, so while it's possible that this might be a problem on some installations of PHP 7.0 and 7.1, I doubt this problem will affect us enough to try and pin down the exact version numbers; hence just removing it.

Reply:
It turns out I get different results between my local PHP 5.6.40 and the 5.6.40 used by GHA. My local works the same as PHP 7+, but GHA has a different result. I restored this work-around but loosened it further, and the work-around only applies to older versions, so in the future we'll just be able to delete this section.

Reply:
Should we add an `@todo` comment to remove it once B requires 7.1+? With exact versions when that odd bug was fixed in php?

> My local works the same as PHP 7+, but GHA has a different result.

Interesting, the action uses php 5.6 from the Sury repo (AFAIK). Where does your local version come from?

Reply:
> Should we add an `@todo` comment to remove it once B requires 7.1+?

I think that at the very least a `@todo` 👍🏼  ...however, since the minimum version is specified via `BACKDROP_MINIMUM_PHP` and we might miss the `@todo` when we update it, I'd suggest to add a conditional check here for `version_compare(BACKDROP_MINIMUM_PHP, '7.1.12', '>=')` and throw a `watchdog` message. That is more likely to be caught and/or reported by people, and that in turn will remind us to actually do the `@todo` 😉